### ISSUE TITLE
Maintenance help wanted?

### ISSUE THREAD

**Issue by leipert** [JedWatson/react-codemirror#125]
Hey there.

We tried updating react-codemirror to the newest version, unfortunately it broke for us, as for other users. See e.g. #106 and #107.
 
I know it is hard to maintain open source projects, we (4 team mates and I at @eccenca) would like to offer to help maintaining this project.

What do you think about it?

/cc other active users:
@skidding @timnew @steveluscher

**Comment by ovidiuch** [comment]
I have no authority here, but it seems like any OSS author's dream to be offered this sort of help. @JedWatson is your man.

**Comment by leipert** [comment]
Yeah I know, I just pinged you (and the others), because maybe you also are able to chime in ;-)

**Comment by scniro** [comment]
@leipert  I commented on various other issues. I threw up [react-codemirror2](https://github.com/scniro/react-codemirror2). I informed everyone I'll be actively maintaining it moving forward. Still unsure why people are opening issues here expecting a reply after months instead of working with me on a successor. I don't know how else to better convey the message or generate more interest...
